packages/docs-site/src/content/docs/resources/terminology.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Taiko-specific Terminology
+description: Resource page detailing relevant rollup terminology that might need explaining.
+---
+
+This page is dedicated to explaining some terms relevant to the protocol that might be a bit hard to understand!
+
+## Bonding-related
+
+- **Liveness Bond**: The bond provided by a block's assigned prover, serving as a commitment to initially prove the first transition and to provide post-contest proofs within the corresponding proving window.
+
+- **Validity Bond**: The bond provided by the actual prover of a proof, signifying their commitment that the proof is indeed correct and that they are ready to withstand a contest.
+
+- **Contest Bond**: The bond provided by the contester.
+
+In layman's terms, the above terms relate to the protocol in the following manner:
+
+- A **Liveness Bond** is deposited when a prover gets assigned a block to prove; when the block gets verified it gets returned to the prover.
```

Comment:
consistency with bulleted lists. will this render actually? need a line break i think.